OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/BackupKeyringParcel.java

```diff
@@ -28,15 +28,12 @@
 
 public class BackupKeyringParcel implements Parcelable {
     public Uri mCanonicalizedPublicKeyringUri;
-    public Passphrase mSymmetricPassphrase;
```

Comment:
@Valodim I will now use CryptoInputParcel for the passphrase. Is this okay or are there reasons to leave this inside the operation parcel?